application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -95,7 +95,7 @@
                         .setRequiredRange(MIN_USE_DURATION, MAX_USE_DURATION),
             new OptionData(OptionType.INTEGER, MAX_AGE_OPTION,
                     "Max age in seconds. Set this to 0 to never expire, default is 1 day", false)
-                        .setRequiredRange(MIN_AGE_DURATION, MAX_AGE_DURATION.getSeconds()));
+                        .setRequiredRange(MIN_AGE_DURATION.getSeconds(), MAX_AGE_DURATION.getSeconds()));
```

Comment:
Just do `0, TimeUnit.DAYS.toSeconds(7)` instead.

Reply:
Changed it now.